src/js/apps/programs/sidebar/action-sidebar_app.js

```diff
@@ -11,9 +11,10 @@ export default App.extend({
   beforeStart() {
     return Radio.request('entities', 'fetch:tags:collection');
   },
-  onStart({ action }, tags) {
+  onBeforeStart({ action }) {
     this.action = action;
-
+  },
+  onStart(options, tags) {
```

Comment:
This appears to fix the issue.

Instead of doing the `if (this.action)` in the `onStop()`. Which stopped the app from borking but caused weirdness in the UI since the `this.action.trigger('editing', false)` doesn't get called correctly.